### PR TITLE
PSP-8552 : FT: Map View- Historical file number search is not working…

### PR DESCRIPTION
**Pull request by eddherrera**
… on the map view

### CODE REVIEW

File: source/frontend/src/components/propertySelector/search/PropertySelectorSearchContainer.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export const PropertySelectorSearchContainer: React.FC<IPropertySelectorSearchCo
 
   useEffect(() => {
     const searchFunc = async () => {
+      console.log(layerSearch);
```
Comment (devinleighsmith) [inline source/frontend/src/components/propertySelector/search/PropertySelectorSearchContainer.tsx]: console.log, remove.

File: source/frontend/src/components/common/mapFSM/useMapSearch.tsx [hunk 2]
```diff
@@ -186,6 +186,66 @@ export const useMapSearch = () => {
...
+    async (filter?: IGeoSearchParams) => {
+      let result: MapFeatureData = emptyFeatureData;
+      try {
+        const loadPropertiesTask = loadPimsProperties(filter);
+
+        let planNumberInventoryData:
+          | FeatureCollection<Geometry, PIMS_Property_Location_View>
+          | undefined;
+        try {
+          planNumberInventoryData = await loadPropertiesTask;
```
Comment (devinleighsmith) [inline source/frontend/src/components/common/mapFSM/useMapSearch.tsx:199]: probably historicalNumberInventoryData

File: source/frontend/src/components/propertySelector/search/PropertySelectorSearchContainer.tsx (hunk 1 above)
Comment (eddherrera) [reply source/frontend/src/components/propertySelector/search/PropertySelectorSearchContainer.tsx]: removed.

File: source/frontend/src/components/common/form/ContactInput/__snapshots__/ContactInputView.test.tsx.snap [hunk 3]
```diff
@@ -217,7 +217,7 @@ exports[`ContactInputView component > renders as expected 1`] = `
                   fill="none"
                 />
                 <path
-                  d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
+                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
```
Comment (asanchezr) [inline source/frontend/src/components/common/form/ContactInput/__snapshots__/ContactInputView.test.tsx.snap:220]: I see a lot of changes in snaps related to icons and I wonder if you have updated your **node_modules** - @eddherrera ? Devin updated react-icons a while back 
It doesn't make sense to see all these changes in your PR as you didn't update any of these screens - did you?

Comment (eddherrera) [reply source/frontend/src/components/common/form/ContactInput/__snapshots__/ContactInputView.test.tsx.snap:220]: I haven't updated node_modules, will do a CI

File: source/frontend/src/components/common/mapFSM/useMapSearch.tsx (hunk 2 above)
Comment (eddherrera) [reply source/frontend/src/components/common/mapFSM/useMapSearch.tsx]: updated

File: source/frontend/src/components/common/form/ContactInput/__snapshots__/ContactInputView.test.tsx.snap (hunk 3 above)
Comment (eddherrera) [reply source/frontend/src/components/common/form/ContactInput/__snapshots__/ContactInputView.test.tsx.snap]: updated

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**